app/src/main/res/layout/activity_baking.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".ui.baking.view.BakingActivity">
+
+    <ProgressBar
+            android:id="@+id/progressBar"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"/>
+
+    <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/baking_recyclerView"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_width="0dp"
+            android:layout_height="0dp"/>
```

Comment:
@KhinThiriSoe width and height values from dimen resource

Reply:
fixed them all except weight and percent value

Reply:
<img width="1179" alt="screen shot 2019-03-06 at 1 16 59 pm" src="https://user-images.githubusercontent.com/10254643/54020346-22824600-41c0-11e9-9135-f5bec10cef05.png">

Reply:
<img width="643" alt="screen shot 2019-03-08 at 4 34 32 pm" src="https://user-images.githubusercontent.com/10254643/54020364-3168f880-41c0-11e9-860c-c978a2cfc2ab.png">